crates/typst-library/src/layout/transform.rs

```diff
@@ -46,14 +46,14 @@ impl Layout for MoveElem {
         let delta = Axes::new(self.dx(styles), self.dy(styles)).resolve(styles);
         let delta = delta.zip_map(regions.base(), Rel::relative_to);
         frame.translate(delta.to_point());
-        Ok(Fragment::frame(frame))
+        return Ok(Fragment::frame(frame));
```

Comment:
Why this change?

Reply:
Seems like a leftover from a previous attempt to make `move` impact layout as well

Reply:
@PgBiel you are absolutely correct, thank you two for pointing it out <3